e2e/testcases/team/user-creation-registrar.spec.ts

```diff
@@ -74,13 +74,10 @@ test.describe.serial('1. Create user -1', () => {
       await page.fill('#password', 'test')
       await page.click('#login-mobile-submit')
 
-      await expect(
-        page
-          .locator('#appSpinner')
-          .or(page.getByText('Welcome to Farajaland CRS'))
-      ).toBeVisible()
+      await expect(page.getByText('Welcome to Farajaland CRS')).toBeVisible({
+        timeout: 60000
```

Comment:
Each `test('', ()=>{})` block has 30s timeout by default. A single assertion to having more timeout than the test itself does not increase the timeout of the test.
```suggestion
        timeout: 30000
```

Reply:
If 30s is not enough, we can mark the test as slow with `test.slow();` to increase the timeout of the test to 90s.